include/http_client.h

```diff
@@ -0,0 +1,22 @@
+#ifndef PROXYSQL_HTTP_CLIENT_H
+#define PROXYSQL_HTTP_CLIENT_H
+
+#include <mongoose.h>
```

Comment:
If I were the reader of the code, I'd like to understand why we are creating a HTTP client in-house. Maybe it pays off to explain a bit that we're already using `mongoose` for the exporting metrics via HTTP part and it pays off to create this very thin layer on top of it.